README.md

```diff
@@ -92,180 +141,66 @@ services:
-
-You can specify the Docker endpoint where an application is installed using a
-context and the `--context` option. If you do not specify one, it will	
-use the currently active context.
+```console
+$ docker context create remote --description "remote cluster" --docker host=tcp://<remote-ip>:<remote-port>
+Successfully created context "remote"
 
-Whenever you define such a context, the installer image will run in the `default`
-context (i.e. on local host). You can use the `--installer-context` to target
-another context to run the installer image.
+$ docker context ls
+NAME                DESCRIPTION                               DOCKER ENDPOINT               KUBERNETES ENDPOINT                ORCHESTRATOR
+default *           Current DOCKER_HOST based configuration   unix:///var/run/docker.sock   https://localhost:6443 (default)   swarm
+remote              remote cluster                            tcp://<remote-ip>:<remote-port>
 
-More examples are available in the [examples](examples) directory.
+$ docker context use remote
+$ docker app run myrepo/hello:0.1.0
+``` 
 
 ## CNAB
 
-Under the hood `docker app` is [CNAB](https://cnab.io) compliant. It generates a
-CNAB from your application source and is able to install and manage any other
-CNAB too. CNAB specifies three actions which `docker app` provides as commands:
-* `install`
-* `upgrade`
-* `uninstall`
-
-Here is an example installing an Application Package, inspect it and
-then uninstalling it:
-```console
-$ docker app install examples/hello-world/example-hello-world.dockerapp --name hello
-Creating network hello_default
-Creating service hello_hello
-
-$ docker app inspect hello
-{
-    "Metadata": {
-        "version": "0.1.0",
-        "name": "hello-world",
-        "description": "Hello, World!",
-        "maintainers": [
-            {
-                "name": "user",
-                "email": "user@email.com"
-            }
-        ]
-    },
-    "Services": [
-        {
-            "Name": "hello",
-            "Image": "hashicorp/http-echo",
-            "Replicas": 1,
-            "Ports": "8080"
-        }
-    ],
-    "Parameters": {
-        "port": "8080",
-        "text": "Hello, World!"
-    },
-    "Attachments": [
-        {
-            "Path": "bundle.json",
-            "Size": 3189
-        }
-    ]
-}
-
-$ docker app uninstall hello
-Removing service hello_hello
-Removing network hello_default
-```
+Docker Apps are Docker’s implementation of the industry standard Cloud Native Application Bundle (CNAB). [CNAB](https://cnab.io/) is an industry specification put in place to facilitate the bundling, sharing, installing and managing of cloud-native apps that are not only made up of containers but also from such things as hosted databases, functions, etc. 
+Docker App is designed to abstract as many CNAB specifics as possible, to provide users with a tool that is easy to use while alleviating the need to bother with the CNAB specification.
```

Comment:
It would be good to add the positive side of the specification: interoperability and a common storage mechanism